demos/interactive_face_detection_demo/utils.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+# pragma once
+
+#include <gflags/gflags.h>
+#include <functional>
+#include <iostream>
+#include <fstream>
+#include <random>
+#include <memory>
+#include <chrono>
+#include <vector>
+#include <string>
+#include <utility>
+#include <algorithm>
+#include <iterator>
+#include <map>
+
+#include <inference_engine.hpp>
+
+#include <samples/common.hpp>
+#include <samples/slog.hpp>
+
+#include <ie_iextension.h>
+#ifdef WITH_EXTENSIONS
+#include <ext_list.hpp>
+#endif
+
+#include <opencv2/opencv.hpp>
+
+// -------------------------Generic routines for detection networks-------------------------------------------------
+
+struct FaceDetectionResult {
+    int label;
+    float confidence;
+    cv::Rect location;
+};
+
+struct AgeGenderResult {
+    float age;
+    float maleProb;
+};
+
+struct HeadPoseResults {
+    float angle_r;
+    float angle_p;
+    float angle_y;
+};
+
+struct Avg {
+    struct Elapsed {
+        explicit Elapsed(double ms) : ss(ms/1000.), mm(static_cast<int>(ss)/60) {}
+        const double ss;
+        const int    mm;
+    };
+
+    using MS = std::chrono::duration<double, std::ratio<1, 1000>>;
+    using TS = std::chrono::time_point<std::chrono::high_resolution_clock>;
+    TS started;
+
+    void    start() { started = now(); }
+    TS      now() const { return std::chrono::high_resolution_clock::now(); }
+    double  tick() const { return std::chrono::duration_cast<MS>(now() - started).count(); }
+    Elapsed elapsed() const { return Elapsed{tick()}; }
+    double  fps(std::size_t n) const { return static_cast<double>(n) / (tick() / 1000.); }
+};
```

Comment:
Please use the OMZ's way to measure performance (drop this one)

Reply:
Done
But there is some mismatch against current fps and total fps now 
Current fps is higher then total fps

Working on it

Reply:
What's the result?

Reply:
Now all look pretty good